task_master.go

```diff
@@ -82,6 +82,11 @@ type TaskMaster struct {
 		StateChangesOnly() bool
 		Alert(channel, message string, level AlertLevel) error
 	}
+	TelegramService interface {
+		Global() bool
+		StateChangesOnly() bool
+		Alert(chatId, parseMode, message string, disableWebPagePreview, disableNotification bool) error
+	}
```

Comment:
Can you also add the TelegramService to the `New` method below so cloned task master preserve this service?

I always forget this one ;)